### PR TITLE
Add klueska to test/e2e/node/OWNERS file

### PR DESCRIPTION
**Pull request by klueska**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
This PR ads me as an owner in `/test/e2e/node`
I am already an owner in `/kubelet` and `/test/e2e_node`.
This folder was overlooked when I was first made an approver for `sig-node` changes.

```release-note
NONE
```

### CODE REVIEW

**Review by ehashman — COMMENTED** [review]
/lgtm
/hold

for other approvers... @SergeyKanzhelev PTAL

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

**Pull request merged**
